### PR TITLE
chore(replays): Log oversized segments

### PR DESCRIPTION
**Pull request by cmanallen**
Gives visibility into oversized segments.

### CODE REVIEW

File: src/sentry/replays/usecases/ingest/__init__.py [hunk 1]
```diff
@@ -215,6 +215,22 @@ def recording_post_processor(
     replay_event_bytes: bytes | None,
     transaction: Span,
 ) -> None:
+    if len(segment_bytes) >= 4_000_000:
+        logger.info(
+            # Logging to the sentry.replays.slow_click namespace because
+            # its the only one configured to use BigQuery at the moment.
+            #
+            # NOTE: Needs an ops request if we want to create a new dataset.
```
Comment (billyvg) [inline src/sentry/replays/usecases/ingest/__init__.py:223]: Would it be worth getting this done for "general" logging?

File: src/sentry/replays/usecases/ingest/__init__.py [hunk 2]
```diff
@@ -215,6 +215,22 @@ def recording_post_processor(
     replay_event_bytes: bytes | None,
     transaction: Span,
 ) -> None:
+    if len(segment_bytes) >= 4_000_000:
+        logger.info(
+            # Logging to the sentry.replays.slow_click namespace because
+            # its the only one configured to use BigQuery at the moment.
```
Comment (michellewzhang) [inline src/sentry/replays/usecases/ingest/__init__.py:221]:
```suggestion
            # it's the only one configured to use BigQuery at the moment.
```

**Review by michellewzhang — APPROVED** [review]

File: src/sentry/replays/usecases/ingest/__init__.py [hunk 3]
```diff
@@ -215,6 +215,22 @@ def recording_post_processor(
     replay_event_bytes: bytes | None,
     transaction: Span,
 ) -> None:
+    if len(segment_bytes) >= 4_000_000:
```
Comment (billyvg) [inline src/sentry/replays/usecases/ingest/__init__.py:218]: Any reasoning for this size?

**Review by billyvg — APPROVED** [review]
just some side comments

File: src/sentry/replays/usecases/ingest/__init__.py (hunk 1 above)
Comment (cmanallen) [reply src/sentry/replays/usecases/ingest/__init__.py:223]: Yeah that's basically what we use this namespace for anyway.  Could extract this out into a global and not have to worry about comments like these anymore.

**Pull request merged**